src/components/Inputs/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -15,9 +15,14 @@ export default {
       options: [...DEFAULT_FEATURES],
     },
     onChange: { action: 'Fired onChange fn' },
+    onImageUpload: { action: 'Fired onImageUpload fn' },
   },
   args: {
     features: [...DEFAULT_FEATURES],
+    onImageUpload: async (file: File) => {
+      console.log(file.name);
```

Comment:
Not sure if this console log is intentional 🤔 😂

Reply:
Just for the storybook story 😄 Thought id be nice to see what you 'uploaded' in the console